src/components/product-card/product-card.component.html

```diff
@@ -1,46 +1,50 @@
-
 <div class="card">
-<img src="{{product.imageURL}}"
-    class="card-img-top" alt="spongebob chair" />
-<div class="card-body">
-    <div class="text-center">
-    <h5 class="card-title">{{product.productName}}</h5>
-    <p class="text-muted mb-4">{{product.productDetails}}</p>
+    <img
+        src="{{ product.imageURL }}"
+        class="card-img-top"
+        alt="spongebob chair"
+    />
+    <div class="card-body">
+        <div class="text-center">
+            <h5 class="card-title">{{ product.productName }}</h5>
+            <p class="text-muted mb-4">{{ product.productDetails }}</p>
+        </div>
+        <div>
+            <div class="d-flex justify-content-between">
+                <span>Cost</span
+                ><span>{{ product.productPrice | currency }}</span>
+            </div>
+            <div class="d-flex justify-content-between">
+                <span>Shipping</span
```

Comment:
I wonder if we should take out the shipping and additional fees part of the product-card/product interface. It feels like we won't be able to calculate shipping and fees until you look at an entire cart.

Reply:
Yeah that sounds like a good idea.

Reply:
I'll leave that for another ticket though, would have to change a lot of things.